vnext/Microsoft.ReactNative.Managed.CodeGen/Model/ReactTypes.cs

```diff
@@ -78,22 +81,25 @@ private ReactTypes(Compilation compilation)
             ReactConstantProvider = FindReactNativeManagedType(compilation, "ReactConstantProvider");
             IReactPromise = FindReactNativeManagedType(compilation, "IReactPromise`1");
             ReactPromise = FindReactNativeManagedType(compilation, "ReactPromise`1");
+            ReactPromiseOfVoid = FindReactNativeManagedType(compilation, "ReactPromise");
             JSValue = FindReactNativeManagedType(compilation, "JSValue");
             JSValueWriter = FindReactNativeManagedType(compilation, "JSValueWriter");
             JSValueReader = FindReactNativeManagedType(compilation, "JSValueReader");
             JSValueWriterGenerator = FindReactNativeManagedType(compilation, "JSValueWriterGenerator");
             JSValueReaderGenerator = FindReactNativeManagedType(compilation, "JSValueReaderGenerator");
-            JSValueWriterCodeGen = FindReactNativeManagedType(compilation, "JSValueWriterCodeGen`1");
-            JSValueReaderCodeGen = FindReactNativeManagedType(compilation, "JSValueReaderCodeGen`1");
+            JSValueWriterOf = FindReactNativeManagedType(compilation, "JSValueWriterOf`1");
+            JSValueReaderOf = FindReactNativeManagedType(compilation, "JSValueReaderOf`1");
```

Comment:
Ah now I get that you 'merged' the two...
I originally seperated so codegen would win because reflection based could sometimes 'double' index types around inheritence or generic types like List<T> ...

Reply:
Thanks! The issue was that the compile-time code was often was not used. I found it to be the simplest fix.